tests/extras/fastapi/test_fastapi.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+pipeline:
+  name: test_fastapi
+
+  default_node_settings:
+    num_cpus: 0.5
+
+  nodes:
+    fastapi:
+      class: aineko.extras.FastAPI
+      node_params:
+        app: tests.extras.fastapi.main:app
+      inputs:
```

Comment:
Can you explain why this is reading and writing to the same topic?

Reply:
There is a endpoint to produce to that topic, and other endpoints to read from the same topic

Reply:
Sorry for some reason I thought this behavior would cause an infinite loop

Reply:
I was concerned about this as well. It might be something we could discuss in a 'best practices' section of documentation. For this fastapi case, it seems to be fine, but for more general node cases, users should be very careful about reading from and writing to the same topic (depending how the dataset is processed within the node)